examples/mavtcpsniff.py

```diff
@@ -13,6 +13,7 @@
 Copyright Sept 2012 David "Buzz" Bussenschutt
 Released under GNU GPL version 3 or later
 '''
+from __future__ import print_function
 
 import sys, time, os, struct
```

Comment:
sys, os, and struct are imported but unused.  It would be useful to run flake8 or autopep8 on this file because the spacing was a bit messed up.